lib/install.go

```diff
@@ -92,91 +91,91 @@ func (i *Install) Run() error {
 		}
 	}
 
-	err = i.DownloadNewBinary()
+	err = i.downloadNewBinary(ctx)
 	if err != nil {
 		return err
 	}
 
-	if !i.NoCheck {
+	if !i.noCheck {
 		stump.Log("binary downloaded, verifying...")
-		err = test.TestBinary(i.TmpBinPath, i.TargetVers)
+		err = test.TestBinary(i.tmpBinPath, i.targetVers)
 		if err != nil {
 			return err
 		}
 	} else {
 		stump.Log("skipping tests since '--no-check' was passed")
 	}
 
-	err = i.MaybeStash()
+	err = i.maybeStash()
 	if err != nil {
 		return err
 	}
 
-	err = i.SelectGoodInstallLoc()
+	err = i.selectGoodInstallLoc()
 	if err != nil {
 		return err
 	}
 
-	stump.Log("installing new binary to %s", i.InstallPath)
-	err = InstallBinaryTo(i.TmpBinPath, i.InstallPath)
+	stump.Log("installing new binary to %s", i.installPath)
+	err = InstallBinaryTo(i.tmpBinPath, i.installPath)
 	if err != nil {
 		// in case of error here, replace old binary
 		stump.Error("Install failed: ", err)
 
 		return err
 	}
 
-	err = i.postInstallMigrationCheck()
+	err = i.postInstallMigrationCheck(ctx)
 	if err != nil {
 		stump.Error("Migration Failed: ", err)
 		return err
 	}
 
-	i.Succeeded = true
+	i.succeeded = true
 	return nil
 }
 
-func (i *Install) RevertOnFailure() {
-	if i.Succeeded {
+func (i *Install) revertOnFailure() {
+	if i.succeeded {
 		return
 	}
 
 	stump.Log("install failed, reverting changes...")
 
-	if i.CurrentVers != "none" && i.InstallPath != "" {
-		revertOldBinary(i.InstallPath, i.CurrentVers)
+	if i.currentVers != "none" && i.installPath != "" {
+		revertOldBinary(i.installPath, i.currentVers)
 	}
 }
 
-func (i *Install) MaybeStash() error {
-	if i.CurrentVers != "none" {
+func (i *Install) maybeStash() error {
+	if i.currentVers != "none" {
 		stump.Log("stashing old binary")
-		oldpath, err := StashOldBinary(i.CurrentVers, false)
+		oldpath, err := StashOldBinary(i.currentVers, false)
 		if err != nil {
 			if strings.Contains(err.Error(), "could not find old") {
 				stump.Log("stash failed, no binary found.")
-				stump.Log(util.BoldText("this could be because you have a daemon running, but no ipfs binary in your path."))
+				stump.Log("** this could be because you have a daemon running, but no ipfs binary in your path. **")
```

Comment:
Why this change? Didn't like the one off use of control characters here?

Reply:
Escape characters do not parse well in logs or display on some types of terminals.

In a conversation with @Stebalien he mentioned that we probably should not be using stump or other non-standard loggers anywhere.

Reply:
fair enough, control characters seem like they should probably be used only if there's a bunch of value (e.g. progress bars, etc.)